.github/workflows/contributor-pr.yml

```diff
@@ -33,9 +33,9 @@ jobs:
         with:
           script: |
             if (context.payload.pull_request && context.payload.pull_request.head.repo.fork) {
-                core.setOutput('sys-prop-args', '-DagreePublicBuildScanTermOfService=yes -DcacheNode=us')
+                core.setOutput('sys-prop-args', '-DagreePublicBuildScanTermOfService=yes -DcacheNode=us -Dscan.tag.GitHubActions')
```

Comment:
We already seem to add the `GH_ACTION` tag. Is there a reason to introduce a new tag? https://ge.gradle.org/s/jt3m4urslkcts

Reply:
Yes, removed.